studio/views.py

```diff
@@ -40,7 +42,7 @@ def has_permission(self, request, view):
             app_instance = AppInstance.objects.get(
                 parameters__contains={'release': release})
             project = app_instance.project
-        except:
+        except:  # noqa: E722
```

Comment:
May be we could make it a TODO as well?

```suggestion
        # TODO: Make it an explicit exception. At least catch `Exception`
        except:  # noqa: E722
```